lib/renewablepotentialslib/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""Top-level package of renewablepotentialslib."""
+
+__version__ = "0.1.0" # Additionally defined in setup.py.
+
+# from https://epsg.io/3035
+EPSG_3035 = "EPSG:3035"
```

Comment:
Does it make sense to have a submodule `geo` here, like we do in `euro-calliope`? It only makes sense if there are several things that do not require geo dependencies. If everything, or almost everything, requires geo dependencies, never mind.

Reply:
Yeah, both the the `conversion` and `shape_utils` packages have geo dependencies, so I see no reason to have a submodule.